x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -308,6 +308,10 @@ principal:: _(Required)_
     against this realm.
     The `principal` appears in places such as the {es} audit logs.
 
+NOTE: You can select to map the SAML `NamedID` value or any other SAML attribute value to the `principal` user
+property. Keep in mind, however, that if a SAML attribute is mapped, the <<saml-logout, Single Logout>> functionality is
+not available.
+
```

Comment:
Is this true? It shouldn't be.
We need to receive a NameID from the IdP, but I don't think we need to map it to `principal`.

Reply:
You're right, I jumped the gun on this without looking at the code closely